pkg/block/block.go

```diff
@@ -183,6 +183,7 @@ type Partition struct {
 	Type       string `json:"type"`
 	IsReadOnly bool   `json:"read_only"`
 	UUID       string `json:"uuid"` // This would be volume UUID on macOS, PartUUID on linux, empty on Windows
+	FSLabel    string
```

Comment:
missing json tags - should probably add `json:"fs_label"` or perhaps we want to go as far as `json:"filesystem_label"`? (I think the former is good)

Reply:
yep, didnt want to add anything here yet as the values can change. Will make sure to add the json tags once we agreed what fields goes where :+1:

Reply:
Fine for me, thanks!

Reply:
@Itxaka @fromanirh OK, good with me to have a separate field then :) Can we name it `FilesystemLabel` and `json:"filesystem_label"` though?

Reply:
you wish is my command, behold a new commit!